src/script/app.js

```diff
@@ -0,0 +1,38 @@
+// Hidden navbar
+let lastKnownScrollPosition = 0;
+
+document.addEventListener("scroll", function () {
+  if (window.scrollY > lastKnownScrollPosition) {
+    document.querySelector(".nav").classList.add("nav_hidden");
+  } else {
+    document.querySelector(".nav").classList.remove("nav_hidden");
+  }
+  lastKnownScrollPosition = window.scrollY;
+});
+
+// Newsletter
+const text = document.querySelector(".text");
+const email = document.querySelector(".form_email");
+const button = document.querySelector(".form_button");
+let regExp = /^[^ ]+@[^ ]+\.[a-z]{2,3}$/;
+
+email.addEventListener("input", function () {
+  if (email.value.match(regExp)) {
+    text.innerHTML = "Your address email in valid";
```

Comment:
Generally speaking setting `innerHTML` should not be the default choice, in this case you don't even need that, you could just attach some text with `textContent` property, however be aware that setting `innerHTML` from untrusted sources like user may lead to `XSS` attack.